libnl-route-3.sym

```diff
@@ -1286,4 +1286,18 @@ global:
 	rtnl_link_bridge_set_vlan_protocol;
 	rtnl_link_bridge_set_vlan_stats_enabled;
 	rtnl_link_inet6_get_conf;
+	rtnl_nh_alloc_cache;
+	rtnl_nh_alloc;
+	rtnl_nh_clone;
+	rtnl_nh_put;
+	rtnl_nh_get;
+	rtnl_nh_compare;
+	rtnl_nh_dump;
+	rtnl_nh_set_gateway;
+	rtnl_nh_get_gateway;
+	rtnl_nh_set_fdb;
+	rtnl_nh_set_fdb;
+	rtnl_nh_get_group_entry;
+	rtnl_nh_get_group_size;
+	rtnl_nh_get_id;
```

Comment:
please sort the lines lexicographically.

Reply:
done